teacher.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unkown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+end
+
+# teacher = Teacher.new('Math')
+# puts teacher.can_use_services?
```

Comment:
- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code, this will make your codebase look neat and readable 😉

Reply:
@prolajumokeoni Thank you